src/main/java/com/nekomaster1000/infernalexp/access/SoulFireAccess.java

```diff
@@ -0,0 +1,71 @@
+package com.nekomaster1000.infernalexp.access;
+
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.annotation.Nullable;
+
+import com.nekomaster1000.infernalexp.InfernalExpansion;
+
+import net.minecraft.client.renderer.model.ModelBakery;
+import net.minecraft.client.renderer.model.RenderMaterial;
+import net.minecraft.client.renderer.texture.AtlasTexture;
+import net.minecraft.util.ResourceLocation;
+
+public interface SoulFireAccess {
+	public KnownFireTypes getFireType();
+
+	public void setFireType(KnownFireTypes type);
+
+	public static final RenderMaterial LOCATION_SOUL_FIRE_0 = new RenderMaterial(AtlasTexture.LOCATION_BLOCKS_TEXTURE, new ResourceLocation("block/soul_fire_0"));
+	public static final RenderMaterial LOCATION_SOUL_FIRE_1 = new RenderMaterial(AtlasTexture.LOCATION_BLOCKS_TEXTURE, new ResourceLocation("block/soul_fire_1"));
+
+	public static final RenderMaterial LOCATION_FIRE_0_GLOW = new RenderMaterial(AtlasTexture.LOCATION_BLOCKS_TEXTURE, new ResourceLocation(InfernalExpansion.MOD_ID, "block/fire_0_glow"));
+	public static final RenderMaterial LOCATION_FIRE_1_GLOW = new RenderMaterial(AtlasTexture.LOCATION_BLOCKS_TEXTURE, new ResourceLocation(InfernalExpansion.MOD_ID, "block/fire_1_glow"));
```

Comment:
```suggestion
	public static final RenderMaterial LOCATION_GLOW_FIRE_0 = new RenderMaterial(AtlasTexture.LOCATION_BLOCKS_TEXTURE, new ResourceLocation(InfernalExpansion.MOD_ID, "block/fire_0_glow"));
	public static final RenderMaterial LOCATION_GLOW_FIRE_1 = new RenderMaterial(AtlasTexture.LOCATION_BLOCKS_TEXTURE, new ResourceLocation(InfernalExpansion.MOD_ID, "block/fire_1_glow"));
```

Why does the glow fire have a different variable name format than the other two